### DISCUSSION TITLE
Latency High/Low/iqm = what Latency in ookla ?

### DISCUSSION THREAD

**Discussion by masterwishx** [alexjustesen/speedtest-tracker#1426] [Q&A] [answered]
can be confirmed that Latency for Download / Upload is : 

Download/Upload Latency low = Minimum Latency
Download/Upload Latency High = Multi-Server Latency
Download/Upload Latency iqm = Primary Server Latency

as described in https://www.ookla.com/resources/guides/speedtest-methodology#latency

**Comment by masterwishx** [comment]
or seems its Jitter not Latency 
Minimum Jitter,Primary Server Jitter,Multi-Server Jitter

**Comment by masterwishx** [comment]
@alexjustesen can you please confirm what exacly values meaned in speedtest-tracker ?
As i need it for my grafana dashboard https://grafana.com/grafana/dashboards/17808-speedtest-tracker-v2-influxdbv2/

**Reply by alexjustesen** [reply]
I'd link back to the docs so you don't create any duplication that needs to be managed: https://docs.speedtest-tracker.dev/other/data-dictionary

This section of the docs could use more information so always happy to review a PR.

**Reply by masterwishx** [reply]
Sure ,but i asked if you can confirrm about new values of data :  its really latency or jitter ?

![image](https://github.com/alexjustesen/speedtest-tracker/assets/28630321/9f21fbcb-0602-4b2c-be75-e7ad51f3473f)

but in ookla in Quality of service (QoS) measurements wrote about jitter :  

![image](https://github.com/alexjustesen/speedtest-tracker/assets/28630321/5d07bf85-826c-46da-8bb6-80ad02820c5a)

or its really latency ? 

![image](https://github.com/alexjustesen/speedtest-tracker/assets/28630321/74ec713b-62c9-4a64-af70-f92d514ec07b)

**Reply by alexjustesen** [reply, marked as answer]
I'm mobile at the moment but feel free to source dive the charts. Here's a latency example: https://github.com/alexjustesen/speedtest-tracker/blob/main/app%2FFilament%2FWidgets%2FRecentDownloadLatencyChartWidget.php#L58

**Reply by masterwishx** [reply]
Thanks , 
Sorry , confused becose ookla in Quality of service (QoS) measurements wrote about jitter . 
https://www.ookla.com/resources/guides/speedtest-methodology#performance-metrics
